controllers/nodefeaturediscovery_resources.go

```diff
@@ -32,9 +32,13 @@ import (
 	"k8s.io/kubectl/pkg/scheme"
 )
 
+// assetsFromFile is a list where each item in the list contains the
+// contents of a given file as a list of bytes
 type assetsFromFile []byte
 
-// Resources holds objects owned by NFD
+// Resources holds objects owned by NFD. This struct is used with the
+// 'NFD' struct to assist in the process of checking if NFD's resources
+// are 'Ready' or 'NotReady'.
```

Comment:
I'd suggest to cut out the second sentence. It might be used elsewhere (in the future, too).

```suggestion
// Resources holds objects owned by NFD.
```